lake-manifest.json

```diff
@@ -1,37 +1,37 @@
-{"version": 5,
+{"version": 6,
  "packagesDir": "lake-packages",
  "packages":
  [{"git":
-   {"url": "https://github.com/mhuisi/lean4-cli.git",
+   {"url": "https://github.com/leanprover/std4",
     "subDir?": null,
-    "rev": "21dac2e9cc7e3cf7da5800814787b833e680b2fd",
+    "rev": "538a09123ed1080bb7227fb5d133abe0e2183b8d",
     "opts": {},
-    "name": "Cli",
-    "inputRev?": "nightly",
+    "name": "std",
+    "inputRev?": "main",
     "inherited": false}},
   {"git":
    {"url": "https://github.com/gebner/quote4",
     "subDir?": null,
-    "rev": "e75daed95ad1c92af4e577fea95e234d7a8401c1",
+    "rev": "a387c0eb611857e2460cf97a8e861c944286e6b2",
     "opts": {},
     "name": "Qq",
     "inputRev?": "master",
     "inherited": false}},
   {"git":
    {"url": "https://github.com/JLimperg/aesop",
     "subDir?": null,
-    "rev": "1a0cded2be292b5496e659b730d2accc742de098",
+    "rev": "4f8b397237411249930791f1cba0a9d2880a02dd",
     "opts": {},
     "name": "aesop",
     "inputRev?": "master",
     "inherited": false}},
   {"git":
-   {"url": "https://github.com/leanprover/std4",
+   {"url": "https://github.com/mhuisi/lean4-cli.git",
     "subDir?": null,
-    "rev":  "67855403d60daf181775fa1ec63b04e70bcc3921",
+    "rev": "39229f3630d734af7d9cfb5937ddc6b41d3aa6aa",
     "opts": {},
-    "name": "std",
-    "inputRev?": "main",
+    "name": "Cli",
+    "inputRev?": "nightly",
     "inherited": false}},
```

Comment:
This swapped the order of the dependencies around; is the manifest file not stable? Not worth blocking this PR for, but possibly worth a fix in lake for the sake of future diffs.

Reply:
Given that the manifest version changed from 5 to 6 here, I think we should assume it is a side effect of that.

Reply:
@eric-wieser, it seems to be a new Lake feature, reported now as https://github.com/leanprover/lean4/issues/2594.